app/services/research_safety_adapters/scishield_api_client.rb

```diff
@@ -19,14 +19,14 @@ def invalid_response?(email)
 
       http_status_error || certification_data_error
     end
-    
+
     def token
       return @token if @token.present?
       return nil unless keys_present?
 
       now = Time.current.to_i
       payload = {
-        iat: now,
+        iat: now - 2.minutes.to_i, # SciShield Support suggested setting this a couple of minutes into that past to make requests more reliable. There may be an issue with the API related to the time change
```

Comment:
How about making this a setting that can be set via env variable and defaults 0 minutes?
```suggestion
        # SciShield Support suggested setting this a couple of minutes in the past to make requests more reliable. 
        # There may be an issue with the API related to the time change for DST
        iat: now - Settings.scishield.iat_offset.minutes.to_i, 
```